### PR TITLE
New condition types

### PR DESCRIPTION
**Pull request by m1kola**
Required for https://github.com/operator-framework/operator-lifecycle-manager/pull/2942

### CODE REVIEW

File: pkg/operators/v1alpha1/installplan_types.go [hunk 1]
```diff
@@ -239,6 +239,9 @@ const (
 	// BundleLookupPending describes BundleLookups that are not complete.
 	BundleLookupPending BundleLookupConditionType = "BundleLookupPending"
 
+	// BundleLookupFailed describes conditions types for when BundleLookups fail
+	BundleLookupFailed BundleLookupConditionType = "BundleLookupFailed"
```
Comment (m1kola) [inline pkg/operators/v1alpha1/installplan_types.go:243]: Currently defined here:

https://github.com/operator-framework/operator-lifecycle-manager/blob/a538741bf9fab9839b56b7b36ac7a5fd7a557bb1/pkg/controller/bundle/bundle_unpacker.go#L36-L38

File: pkg/operators/v1alpha1/subscription_types.go [hunk 2]
```diff
@@ -111,6 +111,12 @@ const (
 
 	// SubscriptionResolutionFailed indicates that the dependency resolution in the namespace in which the subscription is created has failed
 	SubscriptionResolutionFailed SubscriptionConditionType = "ResolutionFailed"
+
+	// SubscriptionBundleUnpacking indicates that the unpack job is currently running
+	SubscriptionBundleUnpacking SubscriptionConditionType = "BundleUnpacking"
+
+	// SubscriptionBundleUnpackFailed indicates that the unpack job failed
+	SubscriptionBundleUnpackFailed SubscriptionConditionType = "BundleUnpackFailed"
```
Comment (tmshort) [inline pkg/operators/v1alpha1/subscription_types.go:119]: Are these used anywhere?

Comment (m1kola) [reply pkg/operators/v1alpha1/subscription_types.go:119]: @tmshort will be used in https://github.com/operator-framework/operator-lifecycle-manager/pull/2942

**Review by perdasilva — APPROVED** [review]
lgtm

**Review by tmshort — APPROVED** [review]
/lgtm

**Pull request merged**